forge/db/models/BrokerCredentials.js

```diff
@@ -36,6 +36,24 @@ module.exports = {
                     this.setDataValue('settings', JSON.stringify(value))
                 }
             }
+        },
+        topicPrefix: {
+            type: DataTypes.STRING,
+            allowNull: true,
+            default: '["#"]',
+            get () {
+                const rawValue = this.getDataValue('topicPrefix')
+                if (rawValue) {
+                    return JSON.parse(rawValue)
+                } else {
+                    return ['#']
+                }
+            },
+            set (value) {
+                if (value) {
+                    this.setDataValue(this.topicPrefix, JSON.stringify(value))
```

Comment:
```suggestion
                    this.setDataValue('topicPrefix', JSON.stringify(value))
```

This was trying to update a field named '#'

![image](https://github.com/user-attachments/assets/33c4ae49-f0ee-4c05-b8f5-7b92f5eddd52)